src/precompile.jl

```diff
@@ -0,0 +1,15 @@
+using SnoopPrecompile
+
+@info "SnoopPrecompile is analyzing PlanetaryEphemeris.jl code..."
+
+@precompile_setup begin
+    # Starting time of integration
+    jd0 = datetime2julian(DateTime(2000,1,1,12)) 
+    # Number of years 
+    nyears = 2031.0 - year(julian2datetime(jd0))
+
+    @precompile_all_calls begin
+        propagate(1, jd0, nyears, Val(false); dynamics = DE430!, order = 25, abstol = 1.0E-20)
+        propagate(1, jd0, nyears, Val(true); dynamics = DE430!, order = 25, abstol = 1.0E-20)
+    end 
+end 
```

Comment:
Why is this file commented in `src/PlanetaryEphemeris.jl`?

Reply:
As a mentioned before, `propagate` has many sources of invalidations. Unfortunately, precompilation directives do not fix this problem, instead, they just add more (precompilation) time.

Reply:
Can you open an issue mentioning this, which we could then address in the future...

Reply:
Ahh, I commented above regarding this... so I reckon we will add `precompile.jl` as a source file in the repo but will not actually include it when loading PE.jl for the time being?

Reply:
We can keep `precompile.jl` for completeness, however, I would not include it in `PlanetaryEphemeris.jl` since it only increases precompilation time without reducing at all warmup time.